app/models/api/v2.rb

```diff
@@ -0,0 +1,5 @@
+module Api::V2
+  def self.table_name_prefix
+    "api_v2_"
+  end
+end
```

Comment:
The Rails generator added this when I created the model with the api_v2 prefix. [Normally it's the empty string](https://apidock.com/rails/ActiveRecord/Base/table_name_prefix/class), we use api_v2 here to keep our things separate from the existing form models, following on from the way the controllers and other v2 files are in a module.